mobile/src/navigation/course/common/constants/constants.ts

```diff
@@ -31,25 +37,46 @@ const SCREEN_OPTIONS: MaterialTopTabNavigationOptions = {
   tabBarInactiveTintColor: AppColor.TEXT.GRAY_200,
 };
 
-const COURSE_TAB_ITEMS: NavigationItem[] = [
+const COURSE_TAB_ITEMS: CoursesNavigationItem[] = [
```

Comment:
Course navigation should be in separate file, this like navigation/courses/courses.navigation.tsx and your tabs should be in navigation/course/course.navigation.tsx

Reply:
A bit confused )
I already have navigation/courses/courses.navigation.tsx ) 
`COURSES - ADD_COURSE - COURSE `

course.navigation.tsx (tabs)
`COURSE - About / My Mentor/ My Student `

course-stack.navigation.tsx 
`COURSE -  EDIT COURSE - COURSE MODULE`

I should move course-stack.navigation.tsx...or maybe rename it?

Reply:
Move EDIT COURSE and COURSE MODULE to COURSES - ADD_COURSE - COURSE

Reply:
they should be on this level

Reply:
everything else looks good 👍

Reply:
And also we can have only one navigation per folder, even if they are related and name them without navigation type `something-details`, `something-tasks` and not `something-tabs`

Reply:
`Move EDIT COURSE and COURSE MODULE to courses navigation (COURSES - ADD_COURSE - COURSE)`

But what I will do with this ways:

- `Courses Management - COURSE (tap on the title) - EDIT_COURSE  (tap on the edit icon)` => Need refactor to `Courses Management - COURSE - EDIT_COURSE - COURSE MODULE ` ?
- `MY_COURSES - COURSE` => `MY_COURSES - COURSE - EDIT_COURSE  - COURSE MODULE`  ?

Reply:
We can navigate from Courses Management to course navigation like this, it's better to use same screen here
```typescript
navigation.navigate({
  screen: AppScreenName.COURSE,
  params: {
    screen: CourseScreenName.EDIT_COURSE
  }
})
```
Same for `MY_COURSES - COURSE`

Reply:
I mean there is no need to duplicate screens to navigate to them

Reply:
Thanks, didn't know that )

Reply:
When I move `Courses Management -> COURSE -> COURSE MODULE` I can't back to` Courses Management`....just on COURSES or back to  COURSE MODULE. Need I send through screens some params or I did some mistake?

Reply:
If I send some params  through screens, have interesting behavior )

<img src = "https://user-images.githubusercontent.com/82529236/191759223-f269bf25-ad67-488f-a454-72c458b2fcb3.gif" width="40%">

Reply:
Oh, ok, then let's leave it as it is, just give another name to navigation, maybe something like configure-course and move it to another folder